core/testing/go.mod

```diff
@@ -1,11 +1,25 @@
 module cosmossdk.io/core/testing
 
-go 1.23
+go 1.23.1
+
+toolchain go1.23.4
 
 require (
 	cosmossdk.io/core v1.0.0
+	github.com/cometbft/cometbft/api v1.0.0
```

Comment:
can we not add this dep however? even if it is the api only, we should try to have core/testing have no dep on cometbft whatsoever.

Reply:
[clean](https://github.com/cosmos/cosmos-sdk/pull/23355/commits/47bdcde047a783529901c5791db2d236ee1eb0a8)